src/main/java/com/jfrog/ide/common/utils/ProjectsMap.java

```diff
@@ -0,0 +1,53 @@
+package com.jfrog.ide.common.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jfrog.build.extractor.scan.DependenciesTree;
+import org.jfrog.build.extractor.scan.GeneralInfo;
+
+import javax.annotation.Nonnull;
+import java.util.Comparator;
+import java.util.TreeMap;
+
+/**
+ * @author yahavi
+ */
+@SuppressWarnings("unused")
+public class ProjectsMap extends TreeMap<ProjectsMap.ProjectKey, DependenciesTree> {
```

Comment:
I suggest adding a description for this class, explaining what it is used for.
Also, let's add a unit test for the ProjectsMap.